src/stores/console.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+const alt = require('../alt');
+const { clearOutput, output  } = require('../actions/console');
+const ConsoleBuffer = require('../../plugins/editor/console-buffer');
+
+class ConsoleStore {
+  constructor() {
+
+    this.bindListeners({
+      onClearOutput: clearOutput,
+      onOutput: output
+    });
+
+    this.state = {
+      buffer: new ConsoleBuffer(),
```

Comment:
I don't think it makes much sense to have `ConsoleBuffer` be its own thing anymore.  It can just be part of the store.  @2fast2fourier thoughts?